### PR TITLE
GitHub contributors

### PR DESCRIPTION
**Pull request by i-moff**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "session2",
+  "version": "0.0.1",
+  "dependencies": {
+    "live-server": "^1.2.0"
+  },
+  "scripts": {
+    "start": "node_modules/.bin/live-server --watch='public/' --open='public/'"
```
Comment (webwin0) [inline package.json:8]: npm knows about node_modules/.bin/
you can write

```
"start": "live-server --watch='public/' --open='public/'"
```

**Comment by webwin0** [comment]
please use spaces instead of tabs. Github shows strange paddings

File: public/scripts/GithubAPIService.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+/**
+ * Class for making request to Github API
+ */
+class GithubAPIService {
+
+		constructor() {
```
Comment (webwin0) [inline public/scripts/GithubAPIService.js:6]: constructor without params doesn't make sense here. 
Better to use constant API_URL imported from consts.js

File: public/scripts/main.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+								contributors: await githubAPIService.getContributors(result.ownerLogin, result.name)
+						};
+				}
+
+				return result;
+		});
+
+		Promise.all(repositories).then((data) => {
+				$('#json-container').jsonPresenter({
+						json: data,
```
Comment (webwin0) [inline public/scripts/main.js:23]: The response in a little different format that required in the task

File: public/scripts/GithubAPIService.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+		}
+
+		/**
+		 * @param owner
+		 * @param repository
+		 * @returns {Promise<any>}
+		 */
+		async getContributors(owner, repository) {
+				const url = new URL(`${this.API_URL}repos/${owner}/${repository}/contributors`);
+				return fetch(url, { method: 'GET' }).then(response => response.json());
```
Comment (webwin0) [inline public/scripts/GithubAPIService.js:30]: Take care about response with an error. For example when a hit rate limit.

**Comment by webwin0** [comment]
Overall looks fine. Good work!

**Comment by i-moff** [comment]
@webwin0 Thanks ;-)

**Pull request merged**